code/modules/clothing/masks/gasmask.dm

```diff
@@ -397,6 +397,67 @@ GLOBAL_LIST_INIT(clown_mask_options, list(
 		to_chat(user, span_notice("Your Mime Mask has now morphed into [choice]!"))
 		return TRUE
 
+/**
+ * # Stage Mask
+ *
+ * Mime stage mask
+ *
+ * A subtype of the regular mime mask with style choices
+ *
+ */
```

Comment:
i dont really understand the point of the comment, the code makes it clear its a subtype of the mime mask and theres functionally no difference so im confused why its being called out.